stdlib/CMakeLists.txt

```diff
@@ -79,6 +79,9 @@ option(SWIFT_BUILD_TEST_SUPPORT_MODULES
        "Whether to build StdlibUnittest and other test support modules. Defaults to On when SWIFT_BUILD_SDK_OVERLAY is On, or when SWIFT_INCLUDE_TESTS is On."
        "${SWIFT_BUILD_TEST_SUPPORT_MODULES_default}")
 
+set(SWIFT_STDLIB_LLVM_LTO OFF CACHE STRING
```

Comment:
What do you think of `SWIFT_STDLIB_LTO_TYPE` and having the options of `none`, `thin`, and `full`?

Reply:
Question: The frontend values for lto are called "llvm-thin" and "llvm-full", I assume to explicitly point out they're enabling LLVM IR LTO, and not any other form of LTO (future SIL LTO, perhaps). Should we keep the "LLVM" name in the variable? Or in the values?

Reply:
I think that `llvm-thin` and `llvm-lto` for the values is reasonable.  The less we have to munge the values, the better.